2022q4/cloud-init.adoc

```diff
@@ -0,0 +1,23 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst] +
+
+Contact: Mina Galić <me+FreeBSD@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Unfortunately, cloud-init support for operating systems other than Linux is rather poor, and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier 1 platform.
+To remedy the situation, this project aims to bring FreeBSD cloud-init support on par with Linux support.
+The broader plan is to lift support across all BSDs.
+
+The first milestone has been delivered.
+Along with many bug fixes, we now have merged an man:ifconfig[8] parser, which allows us to retrieve all the information of all network devices, similarly to how on Linux this is done by parsing the contents of `/sys/class/net/<dev>/`
+In the coming weeks, this project will align itself with the Azure developers to do some crucial refactoring.
+This will move our new parser further into the main execution paths.
```

Comment:
Clarification: what does "main execution paths" mean here?

Reply:
Currently, the ifconfig parser [is merged](https://github.com/canonical/cloud-init/pull/1779), but mostly remains unused, except in some very cases.
The point of the [refactoring project](https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst) is to move more functionality to this new parser.

Reply:
```suggestion
This will make more main path code use our new parser.
```
looks clearer to me; does it mean what you meant?

Reply:
```suggestion
This will make more main path code use our new parser.
```
looks clearer to me; does it mean what you meant?